woocommerce/woocommerce-android
# Remove Jetpack Installation experiment code

Closes: #8104

### Description
This PR removes the Jetpack Installation experiment code, and rolls out the native installation to all of the users.

### Testing instructions
1. Prepare a non-Jetpack site.
2. Open the app then sign in.
3. In the site picker, click on Add a store
4. Select Connect an existing store.
5. Confirm the native Jetpack installation opens.
